### DISCUSSION TITLE
Optimize your images with R and reSmush.it | One world

### DISCUSSION THREAD

**Discussion by giscus** [dieghernan/dieghernan.github.io#34] [Blog]
# Optimize your images with R and reSmush.it | One world

Introducing the resmush package - Compress the size of your images with R, resmush and reSmush.it

https://dieghernan.github.io/202402_optimize-images-r/

**Comment by yihui** [comment]
Hi Diego, I'd like to thank you so much for 1) the **resmush** package, which looks impressive! 2)  mentioning the alternatives in **xfun** and presenting the clear comparison tables; 3) your generous sponsorship! Just a few days ago I have started a short-term contract job with Swiss Re, and I think I'm fine on the financial side now. I have notified all sponsors and told them that they could cancel their sponsorship. I'll also announce it in a blog post in the near future. I really appreciate the help from you and the community!

**Reply by yihui** [reply]
Oh I forgot one thing regarding the package size problem you mentioned in the post. Last year I introduced this little experimental feature in **knitr** (still in the dev version): https://github.com/yihui/knitr/commit/cf94d53a32473c876b720d71a4450c1c7dda01f0 The goal was exactly to reduce the size of vignette images. I'm not sure how much it could help, though.

**Reply by dieghernan** [reply]
Hi @yihui:

Many thanks for your kind words. I am glad you joined a new project; it was clear to me that you have new opportunities soon. For sure, I would try the new **knitr** function once it is released on CRAN.

Best, Diego
